castervoice/lib/ctrl/user.py

```diff
@@ -35,17 +35,17 @@ class UserContentManager(object):
     def __init__(self):
         # self.rules = self.import_user_dir("get_rule", settings.SETTINGS["paths"]["USER_DIR"] + "/rules")
         # self.filters = self.import_user_dir("get_filter", settings.SETTINGS["paths"]["USER_DIR"] + "/filters")
-        self.caster_dir = os.path.realpath(__file__).rsplit(os.path.sep + "castervoice", 1)[0].replace("\\", "/") + "/"
-        self.user_dir = settings.SETTINGS["paths"]["USER_DIR"] + "/"
+        self.caster_dir = os.path.realpath(__file__).rsplit(os.path.sep + "castervoice", 1)[0].replace("\\", "/")
```

Comment:
I see a variation of this all over CodeBase. Why aren't we utilizing something like `CASTER_ROOT_DIR = "Root castervoice Path"` or/and included as an import?

Reply:
Sheer laziness ^^

Reply:
Good to know. In the near future all go through and try to correct these into something more standardized. I wouldn't worry about it here.